### PR TITLE
feat(docs): add 2020Q1 roadmap

### PR DESCRIPTION
**Pull request by dhmlau**
Based on our initial discussion in https://ibm.ent.box.com/notes/556759029402, I'm creating this PR to capture the 2020 Q1 roadmap.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+- Tooling:
+  - improve existing commands like `import-lb3-models`
+  - add new commands (e.g. import datasources).
+
+### Inclusion of related models
+
+- Include related models with a custom scope
+  https://github.com/strongloop/loopback-next/issues/3453
+
+### From model definition to REST API with no custom repository/controller classes https://github.com/strongloop/loopback-next/issues/2036
```
Comment (dhmlau) [inline docs/ROADMAP.md]: move to stretch goals

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+#### Authentication
+
+- Auth0 example: Create a more realistic example than
+  https://github.com/raymondfeng/loopback4-example-auth0.
+
+#### API Connect / LoopBack 4 Integration
+
+- Spike on getting APIs created from LB4 imported into APIC,
+  https://github.com/strongloop/loopback-next/issues/4115
+
```
Comment (bajtos) [inline docs/ROADMAP.md]: I am proposing to add the following section:

```markdown
#### Create models & REST APIs dynamically at runtime

- Dynamic binding/rebinding of controllers after app start #433
- How to build models, repositories and controllers dynamically at runtime #4296
- Docs for exposing REST API of a Model with no custom classes #2740 (stretch goal)
```

(See the spike https://github.com/strongloop/loopback-next/pull/4235).

**Comment by dhmlau** [comment]
> Do we want to remove 2019Q4 roadmap (and possibly other older sections) as part of this change?

Hm. I thought the intention was to keep a history of the roadmap?  I'm good either way.

**Comment by dhmlau** [comment]
@bajtos, since you're away, I'll merge it for now.  If we think it's better to remove the older roadmap, I can do it in a separate PR. Thanks.

**Pull request merged**